diff --color -ur a/src/core/nginx.h b/src/core/nginx.h
--- a/src/core/nginx.h	2022-02-06 21:54:07.227674619 +0200
+++ b/src/core/nginx.h	2022-02-06 21:56:32.836114481 +0200
@@ -11,7 +11,7 @@
 
 #define nginx_version      1021006
 #define NGINX_VERSION      "1.21.6"
-#define NGINX_VER          "nginx/" NGINX_VERSION
+#define NGINX_VER          "ostrich/" NGINX_VERSION
 
 #ifdef NGX_BUILD
 #define NGINX_VER_BUILD    NGINX_VER " (" NGX_BUILD ")"
diff --color -ur a/src/http/ngx_http_header_filter_module.c b/src/http/ngx_http_header_filter_module.c
--- a/src/http/ngx_http_header_filter_module.c	2022-02-06 21:54:07.235674647 +0200
+++ b/src/http/ngx_http_header_filter_module.c	2022-02-06 21:56:32.836114481 +0200
@@ -46,7 +46,7 @@
 };
 
 
-static u_char ngx_http_server_string[] = "Server: nginx" CRLF;
+static u_char ngx_http_server_string[] = "Server: ostrich" CRLF;
 static u_char ngx_http_server_full_string[] = "Server: " NGINX_VER CRLF;
 static u_char ngx_http_server_build_string[] = "Server: " NGINX_VER_BUILD CRLF;
 
diff --color -ur a/src/http/ngx_http_special_response.c b/src/http/ngx_http_special_response.c
--- a/src/http/ngx_http_special_response.c	2022-02-06 21:54:07.239674660 +0200
+++ b/src/http/ngx_http_special_response.c	2022-02-06 21:58:22.192381819 +0200
@@ -33,7 +33,7 @@
 
 
 static u_char ngx_http_error_tail[] =
-"<hr><center>nginx</center>" CRLF
+"<hr><center>ostrich</center>" CRLF
 "</body>" CRLF
 "</html>" CRLF
 ;
diff --color -ur a/src/http/v2/ngx_http_v2_filter_module.c b/src/http/v2/ngx_http_v2_filter_module.c
--- a/src/http/v2/ngx_http_v2_filter_module.c	2022-02-06 21:54:40.927786230 +0200
+++ b/src/http/v2/ngx_http_v2_filter_module.c	2022-02-06 21:56:32.836114481 +0200
@@ -191,7 +191,7 @@
     ngx_http_core_srv_conf_t  *cscf;
     u_char                     addr[NGX_SOCKADDR_STRLEN];
 
-    static const u_char nginx[5] = "\x84\xaa\x63\x55\xe7";
+    static const u_char nginx[7] = "\x85\x3a\x13\x61\x89\x3f";
 #if (NGX_HTTP_GZIP)
     static const u_char accept_encoding[12] =
         "\x8b\x84\x84\x2d\x69\x5b\x05\x44\x3c\x86\xaa\x6f";
@@ -517,7 +517,7 @@
             pos = ngx_http_v2_write_header_str("server", NGINX_VER_BUILD);
 
         } else {
-            pos = ngx_http_v2_write_header_str("server", "nginx");
+            pos = ngx_http_v2_write_header_str("server", "ostrich");
         }
     }
 
diff --color -ur a/src/http/v3/ngx_http_v3_filter_module.c b/src/http/v3/ngx_http_v3_filter_module.c
--- a/src/http/v3/ngx_http_v3_filter_module.c	2022-02-06 21:54:07.243674674 +0200
+++ b/src/http/v3/ngx_http_v3_filter_module.c	2022-02-06 21:56:32.836114481 +0200
@@ -185,7 +185,7 @@
             n = sizeof(NGINX_VER_BUILD) - 1;
 
         } else {
-            n = sizeof("nginx") - 1;
+            n = sizeof("ostrich") - 1;
         }
 
         len += ngx_http_v3_encode_field_lri(NULL, 0,
@@ -368,8 +368,8 @@
             n = sizeof(NGINX_VER_BUILD) - 1;
 
         } else {
-            p = (u_char *) "nginx";
-            n = sizeof("nginx") - 1;
+            p = (u_char *) "ostrich";
+            n = sizeof("ostrich") - 1;
         }
 
         ngx_log_debug2(NGX_LOG_DEBUG_HTTP, c->log, 0,
